### DISCUSSION TITLE
Add a save button for chats

### DISCUSSION THREAD

**Discussion by St33lMouse** [oobabooga/text-generation-webui#1320] [Ideas]
We could use a save button for chats.

**Comment by Woisek** [comment]
Still no save buttons for saving the chats ... 😐

**Comment by Woisek** [comment]
I really want to insist, that we URGENTLY need a chat save button. Please! 😋🔥

**Reply by ossirytk** [reply]
I think the core llama.cpp has support for conversation cache now, so maybe this will be implemented in the future.

**Reply by Woisek** [reply]
Yes, please! This would be awesome, because some of the conversations are really interesting to save/archive. 😄
